src/components/tooltip/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './tooltip';
```

Comment:
This should be `export { default } from './tooltip.component';` I think

Reply:
I thin it needs to point at the declaration file so `export { default } from './tooltip';` would be right